pkg/services/sqlstore/migrations/ualert/ualert.go

```diff
@@ -276,6 +276,11 @@ func (m *migration) Exec(sess *xorm.Session, mg *migrator.Migrator) error {
 			}
 		}
 
+		folderHelper := folderHelper{
```

Comment:
Again, not sure if this extra struct adds much value

Reply:
if you have better idea, I will be glad to consider it.